src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,55 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { ChangeEventHandler } from 'react';
+import { SearchLink } from './SearchLink';
+
+export const PeopleFilters = ({
+  onSearchChange,
+  sex = '',
+  centuries = [],
+  query = '',
+}: {
+  onSearchChange: ChangeEventHandler<HTMLInputElement>;
+  sex?: string;
+  centuries?: string[];
+  query?: string;
+}) => {
+  const isActive = (key: string, value: string) =>
+    (key === 'sex' && sex === value) || centuries.includes(value);
+
+  const toggleCenturies = (centuryParam: string) => {
+    const newCenturies = centuries.includes(centuryParam)
+      ? centuries.filter(century => century !== centuryParam)
+      : [...centuries, centuryParam];
+
+    return { centuries: newCenturies };
```

Comment:
The `toggleCenturies` function is creating a new object every time it is called, which might lead to unnecessary re-renders if used frequently. Consider optimizing this logic if performance becomes an issue.